D8 core 1382 feedback

sherakama opened the pull request:
# So much to talk about.

Comments forthcoming.

sherakama committed:
Fix the install dependencies format. (#5)

* Fix the install depedencies format.

* Swap core for core_version_requirement in info.yml

sherakama committed:
D8CORE-1698: To Dev (#9)

* D8CORE-1698: To Dev

* Update composer.json

sherakama committed:
D8CORE-1382: Component review.

sherakama committed:
D8CORE-1382: Component review.

sherakama committed:
Added configuraition form to newsletter block and More fixups.

sherakama committed:
fixup! some css clean up.

sherakama committed:
What is this patch?

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
-    "drupal/auto_entitylabel": "3.x-dev",
-    "drupal/entity_print": "2.x-dev",
-    "drupal/views_block_filter_block": "^1.0@beta",
+    "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
```
@boznik can this be beta2 or does it have to be on the latest?

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
-    "drupal/entity_print": "2.x-dev",
-    "drupal/views_block_filter_block": "^1.0@beta",
+    "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
+    "drupal/ds": "^3.5",
```
@boznik added this as a requirement as there is a patch for it.

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
-    "drupal/views_block_filter_block": "^1.0@beta",
+    "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
+    "drupal/ds": "^3.5",
+    "drupal/entity_print": "^2.1",
```
@boznik this changed from a dev version. 2.1 seems to be working but do you need the latest dev for something?

boznik replied:
this is fine

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
+    "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
+    "drupal/ds": "^3.5",
+    "drupal/entity_print": "^2.1",
+    "drupal/page_manager": "dev-4.x#78b492786c841c5d2eb07f6d977275a76b79e46f",
```
Added page manager for the dynamic news page. We can talk about https://github.com/SU-SOE/soe_profile/pull/12 and more updates later.

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
+    "drupal/ds": "^3.5",
+    "drupal/entity_print": "^2.1",
+    "drupal/page_manager": "dev-4.x#78b492786c841c5d2eb07f6d977275a76b79e46f",
+    "drupal/views_block_filter_block": "^1.0",
```
There is a stable release for this now.

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
     "drupal/views_infinite_scroll": "^1.6",
-    "drupal/viewfield": "^3.0@beta",
-    "su-sws/jumpstart_ui": "^8.1.0@alpha"
+    "drupal/viewfield": "^3.0-beta5",
```
being more specific with versions in prep for release.

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
-    "drupal/viewfield": "^3.0@beta",
-    "su-sws/jumpstart_ui": "^8.1.0@alpha"
+    "drupal/viewfield": "^3.0-beta5",
+    "su-sws/jumpstart_ui": "dev-8.x-1.x"
```
Using dev version convention here but we will cut a master branch for the release.

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
   "extra": {
     "enable-patching": true,
     "patches": {
-      "drupal/core": {
```
This patch was denied and will not be incorporated. The culprit for the issue is bad config yaml.

sherakama commented on composer.json:
```diff
@@ -13,27 +13,25 @@
       },
       "drupal/ui_patterns": {
-        "patches": "patches/contrib/ui_patterns_views-preview.patch"
+        "ui_patterns_view_patch": "patches/contrib/ui_patterns_views-preview.patch"
```
@boznik What is this patch and why isn't it on d.o?

boznik replied:
This is so you can see the pattern in the views gui

jbickar commented:
https://www.youtube.com/watch?v=mJ58TVYNFro

sherakama commented on lib/js/stanford_news_list.behavior.js:
```diff
@@ -13,9 +13,8 @@ export default {
     (function ($) {
       $(".collapsable-menu", context).click(function() {
-        $(this, context).toggleClass("show");
+        $(this).toggleClass("show");
```
This is a special case when using the `this` keyword already in a context selector.

sherakama commented on lib/js/stanford_news.behavior.js:
```diff
@@ -15,43 +15,42 @@ export default {
-      $('.news-social-media').prepend('<div class="widget-wrapper-linkedin"><a href="" class="share-linkedin"><img src="' + socialImagePath + '/soe_linkedin_icon_blue.svg" alt="linkedin share"></a></div>');
-      $('.news-social-media').prepend('<div class="widget-wrapper-twitter"><a href="" class="share-twitter"><img src="' + socialImagePath + '/soe_twitter_icon_blue.svg" alt="twitter share"></a></div>');
-      $('.news-social-media').prepend('<div class="widget-wrapper-fb"><a href="" class="share-fb"><img src="' + socialImagePath + '/soe_facebook_icon_blue.svg" alt="facebook share"></a></div>');
+      $('.news-social-media', context).prepend('<div class="widget-wrapper-print"><a href="" class="share-print" onclick=""><img src="' + socialImagePath + '/soe_print_icon_gray.svg" alt="print article"></a></div>');
```
Context is important.

sherakama commented on lib/scss/components/filter-menu/_filter-menu.scss:
```diff
@@ -4,11 +4,13 @@
-    flex: auto;
-    max-width: 100%;
+// Changing the layout for the filered pages.
+@include grid-media-max('lg') {
```
@jenbreese Changed to grid-max and nested selectors.

sherakama commented on lib/scss/components/filter-menu/_filter-menu.scss:
```diff
@@ -45,15 +47,15 @@
         content: "\f1de"; // Placeholder Icon until SU ones in place.
         font-family: 'Font Awesome 5 Free';
-        font-weight: 600; // Need this for the new version of FA. Will go away later.
+        font-weight: $su-font-semi-bold; // Need this for the new version of FA. Will go away later.
```
Most of the font-weights we have use variables from Decanter.

jenbreese replied:
Ok, that's good to know.

sherakama commented on lib/scss/components/filter-menu/_filter-menu.scss:
```diff
@@ -64,8 +66,8 @@
       @include modular-spacing('padding-bottom', 2);
-      @include small-paragraph;
       @include modular-spacing('padding-left', 0);
+      @include small-paragraph;
```
I'm just being pedantic here.

jenbreese replied:
Oh below the spacing? Got it. Done.

sherakama commented on lib/scss/components/news-card/_card--horizontal.scss:
```diff
@@ -1,52 +0,0 @@
-// Horizontal Card
-//
-
-.su-card--horizontal {
```
@boznik @jenbreese I couldn't find where this is being used within this module. Was this just leftover copy pasta from something?

sherakama commented on lib/scss/components/news-card/index.scss:
```diff
@@ -1,8 +0,0 @@
-@charset 'UTF-8';
```
@boznik @jenbreese I couldn't find where this is being used within this module. Was this just leftover copy pasta from something?

jenbreese replied:
Yes I believe so.

sherakama commented on lib/scss/components/news-list-item/_news-list-item.scss:
```diff
@@ -1,10 +0,0 @@
-@charset "UTF-8";
```
This file didn't appear to be included in any compile or scss roll up so I removed it.

sherakama commented on lib/scss/components/news-node/_news-node.scss:
```diff
@@ -44,10 +43,6 @@
     display: flex;
     flex-wrap: wrap;
-
-    > div {
-
-    }
```
@jenbreese no empty selectors allowed.

sherakama commented on lib/scss/components/newsletter/_newsletter-signup.scss:
```diff
@@ -16,10 +16,6 @@
     background-color: $su-color-fog;
     text-align: center;
 
-    //.flex-8-of-12 {
-    //  @include margin(0 auto);
-    //}
```
@jenbreese Please remove all commented code prior to review for delivery.

sherakama commented on src/Plugin/Block/SignupBlock.php:
```diff
@@ -13,16 +15,43 @@
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form = parent::blockForm($form, $form_state);
+
+    $config = $this->getConfiguration();
+
+    $form['form_action'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Stanford Mailing List Subscribe URL'),
+      '#description' => $this->t('Example: https://stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797'),
+      '#default_value' => isset($config['form_action']) ? $config['form_action'] : 'https://stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797',
+    ];
+
+    return $form;
+  }
```
@boznik I added a config form so end users can change the subscribe form. I suspect we want this but need confirmation from you.

sherakama commented on stanford_news.info.yml:
```diff
@@ -1,41 +1,43 @@
 type: module
 description: 'Stanford news items and views.'
-core: 8.x
+core_version_requirement: ^8.8 || ^9
```
@boznik New format going forward. Please don't use the core key any longer.

sherakama commented on stanford_news.info.yml:
```diff
@@ -1,41 +1,43 @@
+core_version_requirement: ^8.8 || ^9
 package: Stanford
-version: 8.x-1.x
+version: 8.x-1.0-dev
```
@boznik 8.x-1.x is not a valid Drupal module version.

sherakama commented on stanford_news.info.yml:
```diff
@@ -1,41 +1,43 @@
+  - views_infinite_scroll:views_infinite_scroll
+  - viewfield:viewfield
+  - views_taxonomy_term_name_depth:views_taxonomy_term_name_depth
+  - page_manager:page_manager
+  - page_manager:page_manager_ui
```
We can talk about this later.

boznik replied:
this is breaking install:

  - page_manager:page_manager
  - page_manager:page_manager_ui

sherakama commented on stanford_news.libraries.yml:
```diff
@@ -66,10 +66,4 @@ newsletter_signup:
   version: VERSION
   css:
     base:
-      dist/css/newsletter-signup.css: {}
```
@boznik @jenbreese This css file wasn't being loaded in this library due to a name change it appears. Please confirm that the change is correct.

sherakama commented on stanford_news.libraries.yml:
```diff
@@ -66,10 +66,4 @@ newsletter_signup:
     base:
-      dist/css/newsletter-signup.css: {}
-
-news_vertical_teaser:
-  version: VERSION
-  css:
-    base:
-      dist/css/news-vertical-teaser.css: {}
```
@jenbreese @boznik 
This library was not being added anywhere from what I could tell. The css file itself was being loaded through the ui_pattern.yml file and that is likely good enough unless you plan to add styles here when the component is not in use.

sherakama commented on stanford_news.module:
```diff
@@ -35,32 +37,20 @@ function stanford_news_theme() {
       $variables['#attached']['library'][] = 'stanford_news/news_list';
     }
   }
-  $variables['attributes']['class'][] = Html::cleanCssIdentifier(_stanford_news_change_characters($variables['base_plugin_id']));
-  $variables['attributes']['class'][] = Html::cleanCssIdentifier(_stanford_news_change_characters($variables['derivative_plugin_id']));
```
@boznik I've got to doublecheck on this but from what I could tell we aren't using the css classes from this code and the functionality is being duplicated in stanford_basic.

sherakama replied:
@boznik assigned.

sherakama commented on stanford_news.module:
```diff
@@ -73,39 +63,3 @@ function stanford_news_preprocess_node(&$variables) {
-/**
- * Implements hook_preprocess_menu().
- */
-function stanford_news_preprocess_menu(&$variables, $hook) {
-  $current_path = \Drupal::request()->getRequestUri();
-
-  if ($variables['menu_name'] !== "main") {
-    return;
-  }
-
-  foreach($variables['items'] as &$item) {
-    $menu_item = [$item];
-    _stanford_news_menu_process_submenu($menu_item, $current_path);
-    $item = $menu_item[0];
-  }
-}
```
@boznik This was altering the main menu only. I removed it as it seemed out of place.

sherakama committed:
Update block--menu-block--news-topics.html.twig

sherakama commented on templates/components/news-banner/news-banner.twig:
```diff
@@ -12,8 +12,8 @@
-    {{ news_banner_image }}
-    {{ news_banner_title }}
-    {{ news_banner_topic }}
+<div {{ attributes }} class="su-news-banner {{ modifier_class }}">
```
@jenbreese @boznik 

I have added the `modifier_class` here because that is the convention we have coming from Decanter but thinking about it now that is probably not necessary. We do use the modifier class for the variants but having both the `{{ attributes }}` variable and other HTML properties like class or ID hard wired on the component can produce duplicate properties. This would be better if it was just the `{{ attributes }}` variable on the outer tag and all css classes were added with `{{ attributes.addClass() }}`

sherakama replied:
@jenbreese assigned.

sherakama commented on templates/components/news-banner/news-banner.ui_patterns.yml:
```diff
@@ -1,10 +1,6 @@
 news-banner:
   label: News Banner
   description: "Banner for news page."
-  variants:
-    default:
-      label: Default
-      description: "Default"
```
This is not needed.

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
- *     added. For example: _target. This acts as the attributes for the link button and headline link as well.
- * - card_cta_label: The clickable text for the CTA link
- * - card_button_label: The text label of the button component.
+ * - news_card_image: (needs description)
+ * - news_card_publishing_date: (needs description)
+ * - news_card_headline: (needs description)
+ * - news_card_dek: (needs description)
+ * - news_card_topics: (needs description)
+ * - news_source: (needs description)
```
@jenbreese @boznik please fill in the descriptions.

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
 #}
-
-{# For variant where whole card is a link, suppress all other links within the component #}
-{%- if "su-card--link" in modifier_class -%}
-  {%- set allow_links = false -%}
-{%- else -%}
-  {%- set allow_links = true -%}
-{%- endif -%}
```
@boznik @jenbreese I removed all of the allow_links logic as I cannot see anywhere this is being used in the component setup. Was this a copy over from the card template that got left?

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
-
-    {# Card Image #}
-    {% if news_card_image is not empty -%}
+<ul {{ attributes }} class="su-news-list {{ modifier_class }}">
```
@jenbreese @boznik The outer most wrapper must have the `{{ attributes }}` variable but when we add that in we open up the potential for duplicate `class` properties. This should be worked on to remove the `class` property from the tag and into `{{ attributes.addClass }}`

sherakama replied:
@jenbreese assigned

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
-    {% if news_card_image is not empty -%}
+<ul {{ attributes }} class="su-news-list {{ modifier_class }}">
+  {# Card Image #}
+  {%- block block_news_card_image %}
```
I added the block to stay in consistency with the other variables as they had them too.

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
+<ul {{ attributes }} class="su-news-list {{ modifier_class }}">
+  {# Card Image #}
+  {%- block block_news_card_image %}
+    {% if news_card_image|render|striptags('<drupal-render-placeholder>') is not empty -%}
```
@jenbreese @boznik Drupal likes to put lots of things into template variables like HTML comments and empty tags. When checking for whether or not something is empty trim it down to see if there is any content.

sherakama replied:
@jenbreese the striptags probably should allow for `<img><picture>`

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
-      {%- if news_card_headline is not empty %}
-        <li class="su-news-list__item">
-          <h2>
-            {%- if news_card_headline_link and allow_links -%}
```
@jenbreese @boznik I removed the allow_links logic as it appeared to be unused. Please confirm.

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
+    {% endif -%}
+  {% endblock -%}
+
+  {# An open field for body content #}
```
This comment is incorrect.

jenbreese replied:
Fixed the comments in this file.

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
-      {%- if news_card_topics is not empty %}
-        <li class="su-news-list__item">
-                    <span>
-                        {%- if news_card_topics_link and allow_links -%}
-                        <a href="{{ news_card_topics_link }}" {{ news_card_topics_attributes }}>
-                            {%- endif -%}
-                          {{- news_card_topics -}}
-                          {% if news_card_topics_link and allow_links -%}
-                        </a>
-                        {%- endif -%}
-                    </span>
```
Formatting and removed the allow_links logic.

sherakama commented on templates/components/news-list/news-item-list.twig:
```diff
@@ -1,97 +1,59 @@
-      {% endif -%}
-    {% endblock -%}
-
-    {#        </div> #}{# /su-card__contents #}
```
Please remove commented out code that is not in use.

sherakama commented on templates/components/news-list/news-list.ui_patterns.yml:
```diff
@@ -38,7 +34,7 @@ news-card:
       preview: "I am a link"
   use: "@news/news-list/news-item-list.twig"
   libraries:
-    - card:
```
This library gets prefixed with all sorts of namespacey stuff so it doesn't really matter but we should be good about naming things.

sherakama commented on templates/components/news-list/news-list.ui_patterns.yml:
```diff
@@ -38,7 +34,7 @@ news-card:
+    - newscard:
         css:
           component:
-            ../../../dist/css/news-card.css: { weight: 1000 }
```
@boznik @jenbreese This appears to be the correct CSS file that should be attached to news item list components. Please confirm.

jenbreese replied:
It was corrected to the right name.

sherakama commented on templates/components/news-list/news-list.ui_patterns.yml:
```diff
@@ -1,10 +1,6 @@
 news-card:
```
@jenbreese @boznik This machine name is not what the rest of it is called. This should be changed everywhere it is being used.

jenbreese replied:
I fixed that. I had to run lando drush cim to see the config changes.

sherakama commented on templates/components/news-vertical-teaser/news-vertical-teaser.twig:
```diff
@@ -12,7 +12,7 @@
  */
 #}
-<div {{ attributes }} class="su-news-vertical-teaser">
+<div {{ attributes }} class="su-news-vertical-teaser {{ modifier_class }}">
```
@jenbreese move everything to `{{ attributes }}`

sherakama commented on templates/components/news-vertical-teaser/news-vertical-teaser.ui_patterns.yml:
```diff
@@ -1,10 +1,6 @@
 news-vertical-teaser:
   label: News Vertical Teaser
   description: "News Vertical Teaser"
-  variants:
-    default:
-      label: Default
-      description: "Default"
```
Not needed.

sherakama commented on templates/components/news-vertical-teaser/news-vertical-teaser.ui_patterns.yml:
```diff
@@ -23,7 +19,7 @@ news-vertical-teaser:
       preview: "Link to taxonomy terms"
   use: "@news/news-vertical-teaser/news-vertical-teaser.twig"
   libraries:
-    - card:
```
Name things

sherakama commented on templates/field--node--su-news-topics.html.twig:
```diff
@@ -1,45 +1,34 @@
-{# Create classes array #}
-{% set classes = []
-%}
-
-{# BEM inspired class syntax: https://en.bem.info/
-   Enable this code if you would like field classes like "article__tags", where article is the content type and field_tags is the field name.
-{% set classes = classes|merge([
-  bundle ~ '__' ~ field_name|replace({'field_' : ''})|clean_class
-]) %}
-#}
-
-{% set attributes = attributes.addClass(classes) %}
```
@boznik @jenbreese none of this logic was doing anything so I removed it all. Please confirm.

sherakama commented on templates/field--node--su-news-topics.html.twig:
```diff
@@ -1,45 +1,34 @@
 {% set title_attributes = title_attributes.addClass(label_display == 'visually_hidden' ? 'visually-hidden') %}
 
 {% if label_hidden %}
-    {% if multiple %}
-        <div{{ attributes }}>
-            {% for item in items %}
-                <span{{ item.attributes }}>{{ item.content }}</span>
-            {% endfor %}
-        </div>
-    {% else %}
-        {% for item in items %}
-            <span{{ attributes }}>{{ item.content }}</span>
-        {% endfor %}
-    {% endif %}
-{% else %}
```
Spaces 2 x 2.

sherakama commented on templates/node--stanford-news.html.twig:
```diff
@@ -5,17 +5,6 @@
   'centered-container'
 ] %}
 
-{# BEM inspired class syntax: https://en.bem.info/
-   Enable this code if you would like node classes like "article article--layout-teaser", where article is the content type and teaser is the view mode.
-{% set classes = classes|merge([
-  node.bundle|clean_class,
-  view_mode ? node.bundle|clean_class ~ '--layout-' ~ view_mode|clean_class
-]) %}
-{% set title_classes = [
-  node.bundle|clean_class ~ '__title'
-] %}
-#}
```
Please don't leave commented out code in files.

sherakama commented on templates/node--stanford-news.html.twig:
```diff
@@ -37,55 +26,57 @@
 
       {% if not node.published %}
-        <p class="node--unpublished">{{ 'Unpublished'|t }}</p>
+        {%
+          set data = {
+          'attributes': create_attribute({'class': 'su-alert su-alert--warning'}),
+          'alert_text': "This page is currently unpublished and not visible to the public."|t,
+        }
+        %}
+        {% include "@jumpstart_ui/components/alert/alert.html.twig" with data %}
```
I copied this over from the work we did on stanford_page.

sherakama commented on templates/signup-block.html.twig:
```diff
@@ -7,19 +7,19 @@
-          <h2 class="su-signup-form__heading">Get the latest Research and Ideas</h2>
-          <p>Subscribe to our biweekly email</p>
+          <h2 class="su-signup-form__heading">{{ "Get the latest Research and Ideas"|t }}</h2>
+          <p>{{ "Subscribe to our biweekly email"|t }}</p>
```
@boznik @jenbreese Raw strings need to be translated in twig files.

sherakama commented on webpack.config.js:
```diff
@@ -51,11 +51,9 @@ var webpackConfig = {
   entry: {
     "news-node.behaviors":  path.resolve(srcJS,   "news-node.behaviors.js"),
     "news-node":            path.resolve(srcSass, "components/news-node/index.scss"),
-    "news-card":            path.resolve(srcSass, "components/news-card/index.scss"),
     "news-list-item":       path.resolve(srcSass, "components/news-list-item/index.scss"),
     "news-list.behaviors":  path.resolve(srcJS,   "news-list.behaviors.js"),
-    "newsletter":           path.resolve(srcSass, "components/newsletter/index.scss"),
-    "news-assets":          path.resolve(srcSass, "components/newsletter/index.scss")
```
@boznik @jenbreese I removed what looked like files that were not being included anywhere. Please confirm.

jenbreese replied:
We aren't using those. Should I resolve?

sherakama commented on src/Plugin/Block/SignupBlock.php:
```diff
@@ -13,16 +15,43 @@
+    $form['form_action'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Stanford Mailing List Subscribe URL'),
+      '#description' => $this->t('Example: https://stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797'),
```
DO NOT DEFAULT TO SOE's

boznik replied:
how about this?

sherakama commented on templates/components/news-banner/news-banner.ui_patterns.yml:
```diff
@@ -23,7 +19,7 @@ news-banner:
+    - banner:
         css:
           component:
             ../../../dist/css/news-banner.css: { weight: 1000 }
```
Remove the weight, please.

sherakama replied:
@jenbreese Can you remove the weights in the ui_pattern.yml files and confirm that the css is still being applied correctly. Thank you.

sherakama commented on templates/signup-block.html.twig:
```diff
@@ -7,19 +7,19 @@
   <div class="mailchimp-magazine-block flex-8-of-12">
     <div id="mc_embed_signup">
-      <form action="//stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797" class="validate" id="mc-embedded-subscribe-form" method="post" name="mc-embedded-subscribe-form" novalidate="" target="_blank">
+      <form action="{{ configuration.form_action|default("https://stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797") }}" class="validate" id="mc-embedded-subscribe-form" method="post" name="mc-embedded-subscribe-form" novalidate="" target="_blank">
```
Remove the default form value and change so that no form shows up unless set.

sherakama replied:
@boznik you're assigned.

sherakama committed:
News banner styles.

sherakama committed:
Merge branch 'D8CORE-1382-feedback' of https://github.com/SU-SOE/stanford_news into D8CORE-1382-feedback

sherakama committed:
News banner styles.

jenbreese committed:
D8CORE-1382-feedback: CSS Fixup

boznik committed:
Update stanford_news.info.yml

pulled out page manager as it was breaking install

jenbreese committed:
D8CORE-1382-feedback: Fixing the names for the news list items.

jenbreese committed:
Merging in changes
Merge branch 'D8CORE-1382-feedback' of github.com:SU-SOE/stanford_news into D8CORE-1382-feedback

jenbreese committed:
D8CORE-1382-feedback: fixing the comments

boznik committed:
Update SignupBlock.php

removed soe list as default and not showing block if form action is not set

boznik committed:
updated example to refer to a mailchimp url

sherakama commented on src/Plugin/Block/SignupBlock.php:
```diff
@@ -13,17 +15,48 @@
+        '#theme' => 'signup_block',
+        '#configuration' => $config,
+      ];
+    } else {
```
```suggestion
    } 
    else {
```

sherakama committed:
Update src/Plugin/Block/SignupBlock.php

boznik committed:
Update signup-block.html.twig

jenbreese committed:
D8CORE-1382-feedback: Adding exceptions to the striptags

jenbreese committed:
mergin
Merge branch 'D8CORE-1382-feedback' of github.com:SU-SOE/stanford_news into D8CORE-1382-feedback
